tests/base.py

```diff
@@ -222,14 +223,27 @@ def modify_expected_datatypes(self, expected_records):
             for record in expected_records:
                 for key, value in record.items():
                     self.align_date_type(record, key, value)
+                    self.sort_array_type(record, key, value)
             return
 
         for key, value in expected_records.items(): # Modify a single record
             self.align_date_type(expected_records, key, value)
+            self.sort_array_type(expected_records, key, value)
 
     def align_date_type(self, record, key, value):
         """datetime values must conform to ISO-8601 or they will be rejected by the gate"""
         if isinstance(value, str) and key in ['updated_at']:
             raw_date = self.parse_date(value)
             iso_date = dt.strftime(raw_date,  "%Y-%m-%dT%H:%M:%S.%fZ")
             record[key] = iso_date
+
+    def sort_array_type(self, record, key, value):
+        try:
+            if isinstance(value, list) and value and key in ['ads']:
```

Comment:
Why are you hardcoding 'ads' and 'id' in this method? Can those be inferred from the schema?

Reply:
I'm a bit confused. Is it asking if both value and key is equal to the word 'ads'? Why write `in ['ads']`?

Reply:
Oh sorry no it's saying "value" is truthy (not empty, not None), but then `key in ['ads']` is the same as `key == 'ads'` right? Why not do that?

Reply:
This was a bad copy pasta from adroll. `ads` was a field that was never sorted. At one point I had records that required sorting but based on the last few test runs with the new app/data-set we do not have that issue anymore. I am going to remove this for now and put it back once we end up with records that have list values.

Reply:
Commented the method and put a TODO to use or remove